packages/editor/src/components/editors/InscriptionEditor.tsx

```diff
@@ -42,6 +44,16 @@ const Header = (props: EditorProps) => {
             {data.name.length > 0 && ` - ${data.name}`}
           </div>
         </div>
+
+        {(helpUrl !== undefined || helpUrl !== '') && (
```

Comment:
why '||' ?
if now the helpUrl is '' the or is true so it will render the help icon